modules/op-release-notes-1-8.adoc

```diff
@@ -22,7 +22,7 @@ include::snippets/technology-preview.adoc[]
 
 
 [id="deprecated-features-1-8_{context}"]
-== Deprecated features
+== Deprecated and removed features
 
 * Support for the `tekton.dev/v1alpha1` API version, which was deprecated in {pipelines-title} GA 1.6, is planned to be removed in the upcoming {pipelines-title} GA 1.9 release.
```

Comment:
Could we add more clarity between stopping support and removing the APIs which causes breakages for customers? What I understand from this bullet point is that the alpha CRDs will be removed and if customers don't update their CRs their pipelines will break.